File: clean_flows.py
import json

path = r"C:\Users\Balaji\.node-red\flows.json"
with open(path, 'r', encoding='utf-8') as f:
    data = json.load(f)

bad_types = ['cloudant', 'cloudant out', 'mqtt-broker', 'mqtt in']
cleaned = [n for n in data if n.get('type') not in bad_types]

with open(path, 'w', encoding='utf-8') as f:
    json.dump(cleaned, f, indent=4)

print(f"Cleaned {len(data) - len(cleaned)} nodes.")

File: clean_flows.js
const fs = require('fs');

const flowsPath = 'C:\\Users\\Balaji\\.node-red\\flows.json';
const data = JSON.parse(fs.readFileSync(flowsPath, 'utf8'));

// Filter out problematic IBM Watson and Cloudant node types
const badTypes = ['cloudant', 'cloudant out', 'mqtt-broker', 'mqtt in'];
const cleaned = data.filter(node => !badTypes.includes(node.type));

fs.writeFileSync(flowsPath, JSON.stringify(cleaned, null, 4));
console.log('Successfully cleaned ' + (data.length - cleaned.length) + ' invalid nodes from flows.json.');
